test/sass/mixins/positioning.spec.scss

```diff
@@ -0,0 +1,1386 @@
+@import "../test";
+@import "../../../source/sass/mixins/positioning";
+
+/* stylelint-disable csstools/use-logical  */
+
+@include describe("@mixin inset") {
+
+  @include it("generates correct fallbacks with a single zero value and no dimension") {
+
+    @include assert() {
+
+      @include output {
+        @include inset(0);
+      }
+
+      @include expect {
+        inset: 0;
```

Comment:
This needs fallbacks?

Reply:
Might disallow this usage. It's not conceptually the same as `padding: 0` nor `margin: 0`.